frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/configure.tsx

```diff
@@ -125,7 +125,7 @@ export const EncryptionFormGroup: React.FC<EncryptionFormGroupProps> = ({
   };
 
   return (
-    <FormGroup fieldId="configure-encryption" label="Encryption">
```

Comment:
Removed this since AXE complained about the child elements having nested labels. (Jessie's suggestion)

Reply:
@rexagod this will impact the usual. On this, we step we have two groups - 
1) encryption
2) network
We can pass a custom component in label prop for that so it does not get <label>. 
And its not only one place, if you see below, we have a lot of those.

Reply:
Okay, so I think it'll be better to revert this in that case, so as to avoid a potential regression?

Reply:
Nope, we must fix all of them.
You can pass a custom label.
label=<div></div> or <span></span> whatever suites and does not affect current css.
May be we can use <TextContext>